modules/auxiliary/scanner/msmail/shared/src/msmail/msmail.go

```diff
@@ -0,0 +1,102 @@
+package msmail
+
+import (
+	"crypto/tls"
+	"encoding/base64"
+	"metasploit/module"
+	"net/http"
+	"strings"
+	"time"
+)
+
+func WebRequestCodeResponse(URI string) int {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+	timeout := time.Duration(3 * time.Second)
+	client := &http.Client{
+		Timeout:   timeout,
+		Transport: tr,
+	}
+	req, err := http.NewRequest("GET", URI, nil)
+	req.Header.Set("User-Agent", "Mozilla/5.0 (iPhone; CPU iPhone OS 11_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.0 Mobile/15E148 Safari/604.1")
```

Comment:
This should probably be a MSF configured parameter

Reply:
That's sorta a WIP thing: https://github.com/rapid7/metasploit-framework/pull/9228

Reply:
ah right!